src/Query/index.ts

```diff
@@ -181,6 +181,14 @@ export class Query {
       },
     });
 
+    parts.push({
```

Comment:
this should be across all different facet properties. ie `facets.target_product`  `facets.target_product>atlas>sub_products`. [these (search for facets)](https://cloud.mongodb.com/v2/5bad1d3d96e82129f16c5df3#/clusters/atlasSearch/Search?collectionName=documents&database=search-staging&indexName=default&view=JsonEdit) are [already being programmatically added onto the search index](https://github.com/mongodb/docs-search-transport/blob/main/src/AtlasAdmin/index.ts#L155), so we should be able to add text operators on these fields